tests/backup/backup_helper.go

```diff
@@ -288,7 +289,7 @@ func CreateBackupWithCustomResourceType(backupName string, clusterName string, b
 	return nil
 }
 
-// CreateScheduleBackup creates a scheduled backup
+// CreateScheduleBackup creates a schedule backup
```

Comment:
Minor nit it can `backup schedule` and `scheduled backup`

Reply:
Everywhere else in the code, the term used is "schedule backup" instead of "backup schedule" or "scheduled backup". For instance, functions, vars such as getscheduleBackupUID, scheduleBackupName and so on.

Reply:
We'll need to fix it at all places then.

Reply:
I will try to replace as many occurrences of "schedule backup" as possible.